### ISSUE TITLE
i found out how to make mouse work without any extra software but v fix kinda "gets in the way"

### ISSUE THREAD

**Issue by ddylard** [VFansss/mgs2-v-s-fix#186]
As the title says i discovered how to use perfect mouse and keyboard controls with fps mod (call of duty style lol) using gog release, your fix and first person modded .exe

I did this by using the funtion of "right trigger as mouse" that the game has. The problem is when i save that option and start the game, fps controls work great with the mouse BUT i dont get the benefit of 2k or 4k internal resolution from your fix BUT if i exir rhe game then enter the fix menu, save it, and start the game again, now everything looks better in 2k res BUT the mouse stops working.

I couldnt find a way to make the game work with the mouse and your fix. Of course if i enter the game setup and deactivate and reactivate mouse option and save, mouse works again but no hd graphics.

Anyways im willing to share my keyboard config files so you can try this yourdelf if interested. I know there were lots os people trying to play the fps mod with keyboard and mouse controls doing some really nasty stuff like injecting dll´s to make the mouse work when its not needed at all. Please let me know if tou need anything and thanks dor the fix is great!!

**Comment by ArhumMK** [comment]
You can get native internal resolution by turning up the resolution setting in the original MGS2SConfig.exe. 
But Mouse controls aren't that helpful as it's mapped to the right stick which you mainly use for the sword, looking around in FPS mode still requires the left stick and there's no way to map that to the right stick.

**Comment by Brunokenway07** [comment]
I agree, the GOG version has the best keyboard+mouse configuration for this game, it is the only version that lets you play the First-Person missions smoothly. Unfortunately, as you say V's fix disables GOG's mapping.

Fix this is very simple, but you can't use V's launcher and don't worry, the graphic quality doesn't get affected, after all it doesn't matter if you use V's launcher, the textures are the same.  The big thing that V does is that it lets the game use your graphic card, without V's fix the game uses the integrated graphics which honestly suck.

Now, the first thing you have to do is configure everything in V's launcher, start the game and check that everything is how you want it. Then, close the game and go to the installation folder: Metal Gear Solid 2 Substance\bin use 'MGS2SConfig.exe' to make sure that the graphic and sound settings here are the same you selected in V's launcher, then map the keyboard+mouse and save.

Now open 'Configuration_file.ini' and where it says 'GraphicAdapterName', copy the name of your graphic card it should look like this:
GraphicAdapterName=NVIDIA GeForce GTX 1650

Now go to 'MGS2SSET.ini' and change 'driver=Intel(R) HUD Graphics' for 'driver=NVIDIA GeForce GTX 1650' and save.

To play the game use 'mgs2_sse.exe' instead of V's launcher.

That should do the trick, good luck!

(If you have any questions let me know)
